tests/vmi_networking_test.go

```diff
@@ -594,6 +594,70 @@ var _ = Describe("Networking", func() {
 		})
 	})
 
+	FContext("VirtualMachineInstance with masquerade binding mechanism", func() {
```

Comment:
Wait, you disabled all tests but this one.

Reply:
Yes I know just want to be sure this test pass on the CI before I run it with all the other tests.